Documentation/op-guide/configuration.md

```diff
@@ -92,6 +92,11 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 + default: 0
 + env variable: ETCD_BACKEND_BATCH_INTERVAL
 
+### --backend-bbolt-freelist-type
++ BackendFreelistType is the type of backend boltdb's freelist (it can be array or map).
```

Comment:
BackendBboltFreelistType specifies the type of freelist that boltdb backend uses (array and map are supported types). 

Also probably move this to experimental flag. If the map type works well and gets stable, we might just remove this option and use map as the only option.

Reply:
sounds good